src/components/CounterAlert.js

```diff
@@ -0,0 +1,59 @@
+
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Alert } from 'patternfly-react'
+import style from './sharedStyle.css'
+
+class CounterAlert extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      counter: props.timeout,
+      showAlert: true,
+    }
+
+    this.decrementCounter = this.decrementCounter.bind(this)
+    this.handleDismiss = this.handleDismiss.bind(this)
+    this.timer = props.timeout ? setInterval(this.decrementCounter, 1000) : null
+  }
+
+  decrementCounter () {
+    const state = {
+      counter: this.state.counter - 1,
+    }
+    if (this.state.counter <= 0) {
+      clearInterval(this.timer)
+      this.timer = null
+      this.handleDismiss()
+    }
+    this.setState(state)
+  }
```

Comment:
Should probably use the same `decrementCounter` style as in `SessionActivityTracker`, where the checking code is done in the `setState` callback...
https://github.com/oVirt/ovirt-web-ui/blob/840e846f852971d411b6687f7f1f07bac14a5e79/src/components/SessionActivityTracker.js#L40-L56

```jsx
```